docs/sources/github/README.md

```diff
@@ -6,3 +6,37 @@ There are several connectors available for GitHub:
 - [GitHub non enterprise](github-non-enterprise/README.md) - for non-Enterprise GitHub Cloud (Free/Pro/Teams) organization hosted in github.com.
 - [GitHub Enterprise Cloud](enterprise-server/README.md) - GitHub Enterprise instances hosted by github.com on behalf of your organization.
 - [GitHub Copilot](copilot/README.md) - Copilot data from GitHub Enterprise instances hosted by github.com on behalf of your organization.
+
+
+## Troubleshooting
+
+There are THREE components to auth:
+  - `PRIVATE_KEY` --> filled by you directly in your host's secret/parameter store,
+  - `CLIENT_ID` --> filled by you directly in your host's secret/parameter store,
+  - `github_copilot_installation_id` --> fill in your `terraform.tfvars` file with the installation id of your GitHub App, which is used to generate the `REFRESH_URL` env var on the lambda/cloud function.
+
+Common pitfalls:
+  - creating a OAuth App instead of a GitHub App ; these are NOT the same thing.
+
+### 401 Unauthorized, No JSON body
+
+
+`com.google.api.client.http.HttpResponseException: 401 Unauthorized `
+
+See when the CLIENT_ID value is wrong. Double check this again "Client ID" value shown in your GitHub App's settings page.
+
+
+### 404 Not Found
+`{"message":"Not Found","documentation_url":"https://docs.github.com/rest/reference/apps#create-an-installation-access-token-for-an-app","status":"404"}`
+
+Seen when installation id is wrong. Double check that your URL of the installation of the GitHub App in your organization is something like:
+
+`https://github.com/organizations/Acme-org/settings/installations/12341234`
+
+Then the installation id is `12341234`. You should fill this in your `terraform.tfvars` file as `github_copilot_installation_id`, which is then used to generate the `REFRESH_URL` env var on the lambda/cloud function.
+
+### 404 Not Found
```

Comment:
As per existing docs, 404 can also happened if GH is under IP restriction policies:

> - If you see *404: Not found* in logs please review the *IP restriction policies* that your organization might have; that could cause connections from psoxy AWS Lambda/GCP Cloud Functions be rejected.